pfhedge/features/features.py

```diff
@@ -59,6 +59,28 @@ def __str__(self) -> str:
         return "expiry_time"
 
 
+class UnderlierSpot(StateIndependentFeature):
+    """Spot of the underlier of the derivative."""
+
+    def __str__(self) -> str:
+        return "UnderlierSpot"
```

Comment:
@masanorihirano 

nit: It might be better to make it a snake case for uniformity with the __str__ of the other features?

```suggestion
        return "underlier_spot"
```